core/src/main/java/org/apache/gravitino/metrics/source/RelationDatasourceMetricsSource.java

```diff
@@ -26,15 +26,10 @@
 public class RelationDatasourceMetricsSource extends MetricsSource {
 
   public RelationDatasourceMetricsSource(BasicDataSource dataSource) {
-    super(MetricsSource.GRAVITINO_SERVER_METRIC_NAME);
+    super(MetricsSource.GRAVITINO_RELATIONAL_STORE_METRIC_NAME);
```

Comment:
The current solution may be a workaround. We should find a way to avoid the same issue by improving the metrics framework. @FANNG1 you can also update the code based on my PR.